lib/api/controllers/collectionController.js

```diff
@@ -1,13 +1,26 @@
+/*
+ * Copyright 2017 Kaliop SAS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 'use strict';
 
 const
   _ = require('lodash'),
-  Promise = require('bluebird'),
-  BadRequestError = require('kuzzle-common-objects').errors.BadRequestError,
-  SizeLimitError = require('kuzzle-common-objects').errors.SizeLimitError,
-  assertHasBody = require('./util/requestAssertions').assertHasBody,
-  assertHasIndex = require('./util/requestAssertions').assertHasIndex,
-  assertHasIndexAndCollection = require('./util/requestAssertions').assertHasIndexAndCollection;
+  Bluebird = require('bluebird'),
+  {BadRequestError, SizeLimitError} = require('kuzzle-common-objects').errors,
+  {assertHasBody, assertHasIndex, assertHasIndexAndCollection} = require('./util/requestAssertions');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 101 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)